skimage/measure/_regionprops.py

```diff
@@ -1406,9 +1407,11 @@ def _parse_docs():
     import textwrap
 
     doc = regionprops.__doc__ or ''
-    matches = re.finditer(
-        r'\*\*(\w+)\*\* \:.*?\n(.*?)(?=\n    [\*\S]+)', doc, flags=re.DOTALL
-    )
+    arg_regex = r'\*\*(\w+)\*\* \:.*?\n(.*?)(?=\n    [\*\S]+)'
+    if sys.version_info >= (3, 13):
```

Comment:
What is this change for?

Reply:
After Python 3.13, the docstrings extracted via the `__doc__` attribute now have their initial indentation offsets stripped. Thus this regex now needs to be modified in order for it to work without considering the initial indentation. See: https://docs.python.org/3.13/whatsnew/3.13.html#other-language-changes